### PR TITLE
Providing easy link to mail everyone involved in a workshop

### PR DESCRIPTION
**Pull request by gvwilson**
Instructors frequently need to email everyone involved in a workshop; this provided a one-stop link for doing that.

### CODE REVIEW

File: workshops/views.py [hunk 1]
```diff
@@ -989,6 +989,7 @@ def event_details(request, event_ident):
         'tasks': tasks,
         'todo_form': todo_form,
         'todos': todos,
+        'all_emails' : [t.person.email for t in tasks if (t.person.email and t.person.may_contact)],
```
Comment (pbanaszkiewicz) [inline workshops/views.py]: Maybe something like ↓ would work?

```python
tasks.filter(person__may_contact=True).values_list('person__email', flat=True)
```

**Comment by maneesha** [comment]
Doesn't the "Ask for Attendance" button already do this?

**Comment by gvwilson** [comment]
@maneesha the 'Ask for attendance' button mails the person listed as the contact address to ask them for attendance figures; this mail everyone associated with the event (learners, organizers, helpers, instructors) with a blank subject and body.  I need to do this at least once every time I teach to say, "Hello, welcome to your upcoming workshop, here are some details."

**Comment by chrismedrela** [comment]
What about using a button instead of bootstrap tag?
image
```
{% if all_emails %}<p><a href="mailto:{{ all_emails|join:',' }}" class="btn btn-primary">Mail everyone</a></p>{% endif %}
```

**Comment by gvwilson** [comment]
@pbanaszkiewicz pointed me at the bootstrap tag - I have no strong feeling.

File: workshops/views.py (hunk 1 above)
Comment (chrismedrela) [inline workshops/views.py]: Or even
```
tasks.filter(person__may_contact=True)\
     .exclude(person__email=None)\
     .values_list('person__email', flat=True)
```
to ensure that we exclude empty emails.

Comment (gvwilson) [inline workshops/views.py]: Good catch - done.

**Comment by pbanaszkiewicz** [comment]
Yeah, originally `bootstrap_tag` was used for generating different [color labels](http://getbootstrap.com/components/#labels) for different tags (as in `models.Tag`), not for making links.

**Comment by gvwilson** [comment]
Happy to amend - please advise.

**Comment by pbanaszkiewicz** [comment]
Go with @chrismedrela's button

**Comment by gvwilson** [comment]
Going with @chrismedrela's button.

**Comment by chrismedrela** [comment]
@pbanaszkiewicz, this PR looks fine for me, could I merge it?

**Comment by pbanaszkiewicz** [comment]
@chrismedrela go ahead! :fireworks:

**Pull request merged**